modules/fts/pages/fts--advanced-default-analyzer.adoc

```diff
@@ -0,0 +1,4 @@
+= Default Analyzer
+The default analyzer to be used for this bucket.
+The default value is `standard`.
+A list of available options can be displayed and selected from, by means of the pull-down menu at the right-hand side of the field.
```

Comment:
dropdown* menu

On second thought, I'd recommend dropping this entire line. This level of explanation is not necessary.